public/js/controllers/index.js

```diff
@@ -166,10 +178,23 @@ angular.module('mean.system').controller('IndexController', [
       messageArray.push(message);
       $scope.notifications = messageArray;
       $scope.messageLength = messageArray.length;
+      console.log('@@@@@@@@@@@',messageArray);
     });
 
-    window.onload = () => {
-      $scope.userGames()
+    $scope.tab2 = false;
```

Comment:
I know you have put in much work to make this feature work. Good job. But next time, try to use an appropriate variable/property names. Using names like tab1 and tab2 does not convey much to the reader. Names like playerTab, sendInvitesTab would be more appropriate to use. Good job. Keep it up

Reply:
I have done that, thanks